android_docker_image.md

```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
```

Comment:
Please use capital letter where necessary, Semaphore, Android etc.

Reply:
This sentence is not correct because there is more than one image for Android (the one with Node for React Native people). So we need a different formulation.